src/controllers/users.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+const { User } = require('../models/User.js');
+
+const normalize = ({ id, name }) => {
+  return {
+    id,
+    name,
+  };
+};
+
+const getAllUsers = async(req, res) => {
+  const users = await User.findAll({
+    order: [
+      ['createdAt', 'ASC'],
+    ],
+  });
+
+  res.json(users.map(normalize));
+};
+
+const getOneUser = async(req, res) => {
+  const { userId } = req.params;
+  const findUser = await User.findByPk(userId);
```

Comment:
Let's add some errors if we don't have the required data.
Same about user service

```suggestion
  const { userId } = req.params;
  
  if (!userdId) {
    some error
  }
  
  const findUser = await User.findByPk(userId);
```